src/bridges/zora/ZoraBridge.sol

```diff
@@ -30,27 +30,50 @@ contract ZoraAsk {
     function cancelAsk(address _tokenContract, uint256 _tokenId) external {}
 }
 
+// https://docs.zora.co/docs/smart-contracts/modules/ReserveAuctions/Core/zora-v3-auctions-coreETH
+contract ZoraAuction {
+    function createAuction(
+        address _tokenContract,
+        uint256 _tokenId,
+        uint256 _duration,
+        uint256 _reservePrice,
+        address _sellerFundsRecipient,
+        uint256 _startTime
+    ) external {}
+}
+
 contract ZoraBridge is BridgeBase {
     struct NftAsset {
         address collection;
         uint256 tokenId;
     }
+
     // Holds the VIRTUAL token -> NFT relationship.
     mapping(uint256 => NftAsset) public nftAssets;
 
+    // Holds the VIRTUAL token -> NFT auction relationship.
+    mapping(uint256 => NftAsset) public nftAuctions;
```

Comment:
to make this cheaper, you could make this `mapping (uint256 => bool) public isInAuction`